### PR TITLE
Dailycharts

### PR DESCRIPTION
**Pull request by manuq**

### CODE REVIEW

**Comment by mterzo** [comment]
Can you fix the couple of pep8 style problems you have, long lines and using 2 blank lines between functions.

File: puppetboard/app.py [hunk 1]
```diff
@@ -1109,3 +1110,20 @@ def radiator(env):
...
+@app.route('/daily_reports_chart.json', defaults={'env': app.config['DEFAULT_ENVIRONMENT']})
+@app.route('/<env>/daily_reports_chart.json')
+def daily_reports_chart(env):
+    """Return JSON data to generate a bar chart of daily runs.
+
+    If certname is passed as GET argument, the data will target that
+    node only.
+    """
+    certname = request.args.get('certname')
+    result = get_daily_reports_chart(
```
Comment (mterzo) [inline puppetboard/app.py]: result = get_or_abort(get_daily_reports_chart, ...)
This will allow for nice clean exit if you get a failure calling query()

Comment (manuq) [inline puppetboard/app.py]: Thanks, should I push more commits to this pull-request, will they be rebased before merging? Or should I rebase and force push to this branch?

**Comment by mterzo** [comment]
@manuq I would revise once you get passing tests.

**Comment by manuq** [comment]
@mterzo done.

**Comment by manuq** [comment]
Hi, any chance that this code gets a review?

**Comment by mterzo** [comment]
@corey-hammerton has been busy and no one stepped up for #305.

**Pull request merged**